src/services/io/podio/JEventProcessorPODIO.cc

```diff
@@ -152,6 +153,13 @@ JEventProcessorPODIO::JEventProcessorPODIO() {
             "HcalEndcapNMergedHits",
             "HcalEndcapNClusters",
             "HcalEndcapNClusterAssociations",
+            "HcalEndcapPRawHits",   // this causes premature exit of eicrecon
```

Comment:
Don't add unrelated changes, in particular make sure not to readd lines that were recently removed.